.gitignore

```diff
@@ -27,4 +27,6 @@ storybook-static
 *.tsbuildinfo
 .eslintcache
 .nyc_output
-test-results/
\ No newline at end of file
+test-results/
+
+graphql.config.yml
```

Comment:
what is this file about? :)

Reply:
I think he uses Webstorm :) - https://www.jetbrains.com/help/webstorm/graphql.html#graphql_configuration

Reply:
You're right it's for webstorm ;)